nservicebus/lifecycle/endpointstartandstop.md

```diff
@@ -10,43 +10,45 @@ redirects:
  - nservicebus/lifecycle/iwanttorunwhenbusstartsandstops
 ---
 
-Classes that plug into the startup-shutdown sequence are invoked just after the endpoint instance has been started and just before it is stopped. Use this approach for any tasks that need to execute with the same lifecycle as the endpoint instance.
+Classes that plug into the startup/shutdown sequence are invoked just after the endpoint instance has been started and just before it is stopped. Use this approach for any tasks that need to execute with the same life-cycle as the endpoint instance.
 
 NOTE: The endpoint instance keeps an internal list of instances which need to be stopped but the instances are registered with the `Builder` as Instance Per Call by default. This means that any attempt to resolve a custom instance via dependency injection will result in a new instance and not the one which has been started and which will be stopped.
 
-
 ## Versions 6 and above
 
+The `IWantToRunWhenEndpointStartsAndStops` interface was previously available within the NServiceBus core library as `IWantToRunWhenBusStartsAndStops`. This interface is only available for those using the [NServiceBus.Host](https://www.nuget.org/packages/NServiceBus.Host/). For more information about the NServiceBus.Host, see the [documentation for the host](/nservicebus/hosting/nservicebus-host).
```

Comment:
The `IWantToRunWhenEndpointStartsAndStops` is a new interface. It wasn't previously available in the Core. What was previously available was the `IWantToRunWhenBusStartsAndStops ` interface.